understanding/21/pointer-gestures.html

```diff
@@ -20,14 +20,16 @@ <h2>Intent of this Success Criterion</h2>
 			</figure>
 
 			<p>Examples of path-based gestures include swiping, sliders and carousels dependent on the direction of interaction, and other gestures which trace a prescribed path such as drawing a specific shape. Such paths may be drawn with a finger or stylus on a touchscreen, graphics tablet, or trackpad, or with a mouse, joystick, or similar pointer device.</p>
+     
+      <h3>Disambiguation of Pointer Gestures and Dragging</h3>
 
-			<p>Dragging motions where the movement between the start and end points can go in any direction and is entirely non-prescriptive are not path-based. Dragging motions are covered in Success Criterion 2.5.7 Dragging.</p>
+      <p>Dragging is a motion between the start and end points without an intermediate point which can go in any direction and is entirely non-prescriptive, and therefore not path-based.  Although a user could drag in the same way as a gesture, the difference is that a gesture requires an intermediate point, dragging does not. Path-based gestures are a communication of a particular command to the system. Dragging involves the displacement of an object from one location to another, gestures do not.  Dragging motions are covered in <a href="https://www.w3.org/WAI/WCAG22/Understanding/dragging.html">Success Criterion 2.5.7: Dragging</a>.</p>
```

Comment:
For consistency#s sake we should either use "dragging motion" or "dragging movement" in both 2.5.1 and 2.5.7. I leave it to native speakers to determine the better choice.
I believe this text could ans should be shorter (maybe not quire as short as the one it replaces). I can try to rephrase...